src/Content/Text/BBCode.php

```diff
@@ -1073,14 +1075,21 @@ private static function convertShareCallback(array $attributes, array $author_co
 			default:
 				$text = ($is_quote_share? "\n" : '');
 
+				$authorId = Contact::getIdForURL($attributes['profile'], 0);
```

Comment:
The explicit `0` isn't needed here. It is the default value.

Reply:
Ha, indeed, I'm used to work with the default `-1` value that means `local_user()` so I'm using `0` explicitly whenever.